Process.h

```diff
@@ -138,19 +138,19 @@ typedef struct ProcessClass_ {
 #define Process_sortState(state) ((state) == 'I' ? 0x100 : (state))
 
 
-#define ONE_K 1024L
+#define ONE_K 1024UL
```

Comment:
What about consistency and making this `ULL` too?

Reply:
My thinking is that a type should only be as large as necessary or as practical. For instance:

* Use a `short` when you need to cram things into limited memory, and it's large enough to hold the values in question.

* However, a `short` is often promoted to an `int` automatically anyway, so use an `int` (or often `unsigned int`) even if the values you care about can all be represented by a `short`.

Similarly:

* The constant `1024` can be represented by a plain int type; so, I first used `1024U`, because I just wanted to declare the constant to be unsigned.

* However, the C standard actually only guarantees that an `unsigned int` can represent up to at least 2**16-1, which means you'd only be able to calculate safely a constant as big as `63 * ONE_K`, which is impractical; rather than requiring most uses of the multiplier to specify a size (*e.g.,* `64UL * ONE_K`), I figured it would be much more practical just to declare `ONE_K` with such a declaration instead.

Here are some other notes on the matter:

* This makes the declared type of the constant `ONE_M` also `unsigned long`, but that's not as impractical as making `unsigned int` the declared type of the constant `ONE_K`; indeed, I only had to specify `ULL` on one rather large multiplier (`10000ULL * ONE_M`).

* I actually do not like the idea of using `unsigned long long` at all; though I haven't looked, I'm guessing it introduces a not-insignificant overhead to the calculations on a 32-bit machine; it would be much better to check an input (*e.g.,* the value of `number`) for its size, and then pass it to code that uses `unsigned long long` only if necessary.

    Heck, maybe a 32-bit machine will never show numbers big enough to warrant `unsigned long long`, anyway, and so the offending code should just not be built at all on a 32-bit machine. I don't know enough to make that determination, so this commit is a good fix until someone can tailor the code more tightly to each target architecture.